frontend/fingertips-frontend/components/organisms/PopulationPyramidChartTable/index.tsx

```diff
@@ -66,12 +72,15 @@ const StyleSelectedAreaTableContextDiv = styled('div')({
   'flexGrow': 8,
   'minWidth': DefaultMinimumWidthForTablePanel + 30,
   '& table': {
-    '& td:first-child': {
+    '& td:first-child, th:first-child': {
       minWidth: '90px',
+      textAlign: 'left',
     },
     '& td, th': {
       borderTopColor: GovukColours.MidGrey,
       borderBottomColor: GovukColours.MidGrey,
+      textAlign: 'right',
+      padding: 5,
```

Comment:
Padding and margins should always have units unless you are specifying 0.  Could you put these in please.